action.yml

```
@@ -17,12 +17,9 @@ inputs:
   app-path:
     description: 'The binary file path to upload to wolfia'
     required: true
-  link-description:
-    description: 'Description for the magic link'
-    default: ""
-  comment-on-pr:
-    description: 'Whether to comment the generated magic link on PR'
-    default: true
+  track-id:
+    description: 'The track id to related to the app. You can find the track id in [Wolfia](https://wolfia.com/apps).'
```

```suggestion
    description: 'The track ID of the app you want to release. You can find the track ID in [Wolfia](https://wolfia.com/apps).'
```

(feel free to skip id capitalization, mostly personal pref)